<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>exportKeyPackageDialog</class>
    <widget class="QDialog" name="exportKeyPackageDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>527</width>
                <height>385</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Export As Key Package</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
                <widget class="QDialogButtonBox" name="button_box_">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
            <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                                <widget class="QLabel" name="nameLabel">
                                    <property name="text">
                                        <string>Key Package Name</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="gnerateNameButton">
                                    <property name="text">
                                        <string>Generate Key Package Name</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QLabel" name="nameValueLabel">
                            <property name="text">
                                <string>KeyPackage_0000</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="Line" name="line_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="selectOutputPathLabel">
                                    <property name="text">
                                        <string>Output Path</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="setOutputPathButton">
                                    <property name="text">
                                        <string>Select Output Path</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QLabel" name="outputPathLabel">
                            <property name="text">
                                <string/>
                            </property>
                            <property name="wordWrap">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="Line" name="line">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QLabel" name="passphraseLabel">
                                    <property name="text">
                                        <string>Passphrase</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="generatePassphraseButton">
                                    <property name="text">
                                        <string>Generate and Save Passphrase</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QLabel" name="passphraseValueLabel">
                            <property name="text">
                                <string/>
                            </property>
                            <property name="wordWrap">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="Line" name="line_3">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QCheckBox" name="includeSecretKeyCheckBox">
                            <property name="text">
                                <string>Include secret key (Think twice before acting)</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QCheckBox" name="noPublicKeyCheckBox">
                            <property name="text">
                                <string>Exclude keys that do not have a private key</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="tipsLabel">
                            <property name="text">
                                <string>Tips: You can use Key Package to safely and conveniently transfer your public
                                    and private keys between devices.
                                </string>
                            </property>
                            <property name="wordWrap">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="verticalSpacer">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>40</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>button_box_</sender>
            <signal>accepted()</signal>
            <receiver>exportKeyPackageDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>button_box_</sender>
            <signal>rejected()</signal>
            <receiver>exportKeyPackageDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
